### DISCUSSION TITLE
Sidebar item grouping

### DISCUSSION THREAD

**Discussion by davidlav** [elchininet/custom-sidebar#113] [Ideas]
Hi, I think it would be very handy to have the ability to group sidebar items together, either in collapsible groups in which multiple sections could be expanded at a time, or in some sort of accordion system where only a single group could be expanded at a time.

**Comment by elchininet** [comment]
Hi @davidlav,
Check [my comment here](https://community.home-assistant.io/t/custom-sidebar-manage-home-assistants-sidebar-items-per-user-or-device-basis/665800/14?u=elchininet). There have been discussions to implement that natively in Home Assistant but it has not been accepted/not much track from the community. That would be a big change and it would change radically the code of this plugin, so if that is not requested from an important part of the community it doesn‘t worth the effort of investing time implementing it.
Regards
